lib/block_view_profile.go

```diff
@@ -1005,10 +1005,8 @@ func _verifyEthPersonalSignature(signer, data, signature []byte) error {
 	}
 
 	// Change the data bytes into Ethereum's personal_sign message standard. This will prepend the message prefix and hash
-	// the prepended message using keccak256. We turn data into a hex string and treat it as a character sequence which is
-	// how MetaMask treats it.
-	dataHex := hex.EncodeToString(data)
-	hash, _ := TextAndHash([]byte(dataHex))
+	// the prepended message using keccak256.
+	hash, _ := TextAndHash(data)
```

Comment:
I'm assuming this change is safe since this is a new feature, but basically we don't need to hex encode the data?

Reply:
This was there before because metamask signs data as a string instead of a byte array. Previously metamask would sign the hex encoding of the data, while now we just sign the metamask string.